languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -19232,6 +19232,46 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rule id="SV_AGR_1" name="Subject-verb agreement rule #1" default="temp_off"><!-- experimental -->
+            <antipattern>
+                <token regexp="yes">i|you|we|they</token>
+                <token min="0" postag="RB"/>
+                <token chunk_re="B-VP" postag="VBP?" postag_regexp="yes"/>
+            </antipattern>
+            <antipattern>
+                <token skip="3" postag="SENT_START"/>
+                <token postag="SENT_END"/>
+            </antipattern>
+            <antipattern>
+                <token>there</token>
+                <token>are</token>
+            </antipattern>
+            <pattern>
+                <token postag="SENT_START"/>
+                <token skip="-1" chunk_re=".-NP-singular" max="10">
+                    <exception postag="WP|VBN?|RB|WDT|UNKNOWN" postag_regexp="yes"/>
+                    <exception regexp="yes">th[eo]se|ok|okay|thanks?|yup|will|m?any|we|they|i|you|both|n?either|what|which|all|some|press</exception>
+                    </token>
+                <marker>
+                <token chunk_re="B-VP" postag="VBP?" postag_regexp="yes">
+                    <exception scope="previous" regexp="yes">(some|any)(body|one)|i|you|we|they|this|that</exception>
+                    <exception scope="previous" postag="N.*S|UNKNOWN|WP|PCT|``" postag_regexp="yes"/>
+                    <exception regexp="yes">do|install|curl|renew|remove|create|set|run|make|please|thank</exception>
+                    <exception postag="VBD|MD" postag_regexp="yes"/>
+                    <exception scope="next" postag="PCT"/>
+                    </token>
+                </marker>
+            </pattern>
+            <message>This verb form does not seem to match the subject.</message>
```

Comment:
The message is fine if the subject and the verb are close, as in the first example.

The message is not so good for the third example if the user does not have a good knowledge of grammar. What is the subject? A nice enhancement would be to include the subject 'lady' in the message.